src/screens/SignIn/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+import nimbleLogoWhite from 'assets/images/icons/nimble-logo-white.svg';
+import background from 'assets/images/illustrations/background-sign-in.png';
+import ElevatedButton from 'components/ElevatedButton';
+import TextInput from 'components/TextInput';
+
+export const signInScreenTestIds = {
+  nimbleLogo: 'sign-in__nimble-logo',
+  signInForm: 'sign-in-form',
+  emailLabel: 'sign-in-form__email',
+  emailField: 'sign-in-form__input-email',
+  passwordLabel: 'sign-in-form__password',
+  passwordField: 'sign-in-form__input-password',
+  forgotButton: 'sign-in-form__forgot-button',
+  signInButton: 'sign-in-form__button',
+};
+
+const SignInScreen = (): JSX.Element => {
+  return (
+    <div
+      className="sign-in bg-cover min-h-screen flex flex-col justify-center items-center"
+      style={{ backgroundImage: 'url(' + background + ')' }}
```

Comment:
```suggestion
      style={{ backgroundImage: `url '${background}'` }}
```

Reply:
But can we try [this](https://tailwindcss.com/docs/background-image) out?

Reply:
@nvminhtue ah thanks, I will update it 👏